<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="addPhotos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>149</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    margin-left: 10px;
    border-image: none;
    border-radius: 14px;
    border-style: solid;
    border-width: 1px;
    font-size: 14px;
    line-height: 20px;
    margin-bottom: 0;
    padding: 4px 12px;
    text-align: center;
    background-color: #006DCC;
    color: #FFF;</string>
   </property>
   <property name="text">
    <string>1、添加照片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dispose">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>149</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    margin-left: 10px;
    border-image: none;
    border-radius: 14px;
    border-style: solid;
    border-width: 1px;
    font-size: 14px;
    line-height: 20px;
    margin-bottom: 0;
    padding: 4px 12px;
    text-align: center;
    background-color: #006DCC;
    color: #FFF;</string>
   </property>
   <property name="text">
    <string>2、照片处理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addCSV">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>149</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    margin-left: 10px;
    border-image: none;
    border-radius: 14px;
    border-style: solid;
    border-width: 1px;
    font-size: 14px;
    line-height: 20px;
    margin-bottom: 0;
    padding: 4px 12px;
    text-align: center;
    background-color: #006DCC;
    color: #FFF;</string>
   </property>
   <property name="text">
    <string>3、增加CSV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trainingModel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>149</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    margin-left: 10px;
    border-image: none;
    border-radius: 14px;
    border-style: solid;
    border-width: 1px;
    font-size: 14px;
    line-height: 20px;
    margin-bottom: 0;
    padding: 4px 12px;
    text-align: center;
    background-color: #006DCC;
    color: #FFF;</string>
   </property>
   <property name="text">
    <string>4、模型训练</string>
   </property>
  </widget>
  <widget class="QPushButton" name="faceRecognition">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>149</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    margin-left: 10px;
    border-image: none;
    border-radius: 14px;
    border-style: solid;
    border-width: 1px;
    font-size: 14px;
    line-height: 20px;
    margin-bottom: 0;
    padding: 4px 12px;
    text-align: center;
    background-color: #006DCC;
    color: #FFF;</string>
   </property>
   <property name="text">
    <string>5、人脸识别</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
